src/test/java/com/thealgorithms/strings/IsomorphicTest.java

```diff
@@ -0,0 +1,40 @@
+package com.thealgorithms.strings;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.*;
+
+/**
+ * Isomorphic
+ */
+public class IsomorphicTest {
+
+    @Test
+  public static void main(String[] args) {
```

Comment:
![image](https://user-images.githubusercontent.com/29201949/189057162-86a32629-b9dd-44a1-89e1-9eefec4f3229.png)
Indentation is inconsistent and it's not the entry point